webpack.config.js

```diff
@@ -63,18 +64,47 @@ module.exports = {
         ],
       },
       {
-        test: /\.svg$/,
-        loader: 'file-loader',
-        query: {
-          name: 'static/media/[name].[hash:8].[ext]',
-        },
-      },
-      {
-        test: /\.(woff|woff2|eot)$/,
+        test: /\.(woff|woff2|eot|png|jpe?g|gif)$/,
         use: {
           loader: 'url-loader',
         },
       },
+      {
+        test: /\.svg$/,
+        include: [/src\/svgs\//],
+        use: [
+          {
+            loader: 'svg-sprite-loader',
+          },
+          {
+            loader: 'svgo-loader',
+            options: {
+              plugins: [
+                { removeTitle: true },
+                { removeNonInheritableGroupAttrs: true },
+                { removeUselessStrokeAndFill: false },
+                { collapseGroups: true },
+                { convertColors: { shorthex: false } },
+                { convertPathData: false },
+                {
+                  removeAttrs: {
+                    attrs: '(fill|stroke)',
+                    preserveCurrentColor: true,
+                  },
+                },
+              ],
+            },
+          },
+        ],
+      },
+      // Any non-sprited SVG images or similar use file-loader
+      {
+        test: /\.svg$/,
+        exclude: /node_modules/,
+        include: [],
```

Comment:
TODO: I believe we'll need to "add back" any SVGs we do NOT want pulled in via SVG sprites e.g. anything we want to leave inline or pulled in via css background images etc. etc.